pkg/config/sort_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build unit
```

Comment:
fixuped in this forgotten build tag

Reply:
@warber please re-approve :) 
vet was luckily failing here because this test imported something this also behind the unit build tag